source/v5/mqtt3_to_mqtt5_adapter.c

```diff
@@ -1819,12 +2393,26 @@ void aws_mqtt3_to_mqtt5_adapter_operation_table_remove_operation(
 
     struct aws_mqtt3_to_mqtt5_adapter_operation_base *operation = existing_element.value;
     if (operation != NULL) {
-        aws_mqtt3_to_mqtt5_adapter_operation_destroy(operation);
+        aws_mqtt3_to_mqtt5_adapter_operation_release(operation);
+    }
+}
+
+struct aws_mqtt3_to_mqtt5_adapter_operation_base *aws_mqtt3_to_mqtt5_adapter_operation_release(
+    struct aws_mqtt3_to_mqtt5_adapter_operation_base *operation) {
+    if (operation != NULL) {
+        aws_ref_count_release(&operation->ref_count);
     }
+
+    return NULL;
 }
 
-void aws_mqtt3_to_mqtt5_adapter_operation_destroy(struct aws_mqtt3_to_mqtt5_adapter_operation_base *operation) {
-    (*operation->vtable->destroy_fn)(operation);
+struct aws_mqtt3_to_mqtt5_adapter_operation_base *aws_mqtt3_to_mqtt5_adapter_operation_acquire(
+    struct aws_mqtt3_to_mqtt5_adapter_operation_base *operation) {
+    if (operation != NULL) {
+        aws_ref_count_acquire(&operation->ref_count);
+    }
+
+    return operation;
 }
 
 void aws_mqtt3_to_mqtt5_adapter_operation_reference_adapter(
```

Comment:
Trivial: I think we use acquire and release for referencing and dereferencing. Should we stick to that naming convention for readability? I felt like I knew what was going on in this function from the context but had to check to be sure because it was doing the same thing but named different from the immediately following line up at line 1667 to 1670.